tests/Exclusion/ExclusionManagerTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+declare(strict_types=1);
+
+namespace Rector\Tests\Exclusion;
+
+use Mockery;
+use PhpParser\Node;
+use PHPUnit\Framework\TestCase;
+use Rector\Contract\Exclusion\ExclusionCheckInterface;
+use Rector\Exclusion\ExclusionManager;
+use Rector\Rector\AbstractRector;
+
+final class ExclusionManagerTest extends TestCase
```

Comment:
Looking at this test again... the functionality is already covered well by: https://github.com/rectorphp/rector/pull/2087/files#diff-b4a2ec27faf137b39d3dbd3e8565d3e0

It can be removed completely.

Reply:
It's ready to merge then, great job :+1:

Reply:
Technically not everything is covered in in the other test, like having multiple exclusionChecks or none at all. So removing this would result in losing some test-coverage.

Reply:
Of course, but the code coverage isn't the goal of tests.

Usable and maintainble tests are. In respect of Occam's razor, this adds more complexity than value. Thus should be removed

Reply:
I'm mergin it as it is, so we can test the functionality.
If needed, I'll do change later.